### PR TITLE
Add static_placeholders for more texts

### PR DESCRIPTION
**Pull request by jandd**
This PR adds static placeholder tags for a few more templates used during the attendee registration process.

### CODE REVIEW

File: devday/attendee/templates/attendee/profile.html [hunk 1]
```diff
@@ -128,27 +128,27 @@ <h5>{% trans "Workshops / Sessions" %}:</h5>
...
-                                                           href="{% url "attendee_cancel" event=event.slug %}">{% trans "Cancel Registration" %}</a>
-                                                    {% endif %}
-                                                </div>
-                                            {% endif %}
                                         {% endif %}
                                         {% endcomment %}
+                                        {% if not event.is_started %}
+                                            <div class="btn-group float-left float-sm-right">
+                                                <a class="btn btn-{% if not attendee.badgedata %}warning{% else %}success{% endif %}"
+                                                   href="{% url "edit_badge_data" event=event.slug %}">{% trans "Edit badge data" %}</a>
```
Comment (snoopotic) [inline devday/attendee/templates/attendee/profile.html]: edit badge data

File: devday/attendee/templates/attendee/profile.html [hunk 2]
```diff
@@ -128,27 +128,27 @@ <h5>{% trans "Workshops / Sessions" %}:</h5>
...
+                                                {% if not user.get_speaker %}
+                                                    <a class="btn btn-danger"
+                                                       href="{% url "attendee_cancel" event=event.slug %}">{% trans "Cancel Registration" %}</a>
```
Comment (snoopotic) [inline devday/attendee/templates/attendee/profile.html]: cancel registration

File: devday/attendee/templates/attendee/profile.html (hunk 1 above)
Comment (jandd) [reply devday/attendee/templates/attendee/profile.html]: these are link/button titles and are therefore written like this

File: devday/attendee/templates/attendee/profile.html (hunk 2 above)
Comment (jandd) [reply devday/attendee/templates/attendee/profile.html]: this is also a button title

**Pull request merged**